chore: Minimal set of permissions to deploy che on openshift (#2794)
codenvy/che-docs: modules/administration-guide/pages/permissions-to-install-che.adoc

:_content-type: ASSEMBLY
:description: Permissions to install {prod}
:keywords: administration-guide, installing, permissions
:navtitle: Permissions to install Che
:page-aliases:

[id="permissions-to-install-che"]
= Permissions to install {prod-short}

> This section describes the permissions required to install {prod} on different {kubernetes} clusters.

Reviewer: ```suggestion
Learn about the permissions required to install {prod} on different {kubernetes} clusters.
```
